codesign/README.md

```diff
@@ -0,0 +1,2 @@
+A sample command-line application with an entrypoint in `bin/`, library code
```

Comment:
You should probably fill this in with at least a few lines giving people context on what it is.